finance-api/api-reference.md

```diff
@@ -0,0 +1,288 @@
+</details>
+
+
+### Responses
+<details open="true">
+<summary>HTTP status codes</summary>
+
+|Status code |Description
+|:---- |:----
+|200 OK |Returned when the account balance is successfully fetched.  
+|403 Forbidden |Returned when you are not authorised to fetch the account balance.
+|400 Bad Request |Returned when a required parameter is missing or in a wrong format in the request .  
+</details>
+
+<details open="true">
+<summary>Response attributes</summary>
+
+|Name |Type |Description
+|:---- |:---- |:----
+|data |object|Information about the account balance at a point in time. It's represented by `totalBalance` and `currencyId`.
+|totalBalance |integer |The account balance in the currency's smallest unit. It can be negative, such as when refunds are greater than sales. In the following example, the account balance is -3 SEK. <br/><pre>{ <br/>totalBalance: -300, <br/>currencyId: "SEK" <br/>}</pre>
+|currencyId |string |The currency of the account. For example, `SEK`.
+</details>
+
+
+## Fetch account transactions
+Returns all transactions or transactions of certain types from a merchant's Zettle preliminary or liquid account during a specific period.
+
+```
+GET /organizations/{organizationUuid}/accounts/{accountTypeGroup}/transactions?start={start_time}&end={end_time}
+```
+
+See example [Fetch transactions for a liquid account](#fetch-transactions-for-a-liquid-account).
+
+### Parameters
+
+<details open="true">
+<summary>Parameters</summary>
+
+|Name |Type |In |Required/Optional |Description
+|:---- |:---- |:---- |:---- |:----
+|organizationUuid |string |path |required |Unique identifier for your organization. You can specify the value with one of the following: <br/><ul><li> Use `self` as the value. This will retrieve your organizationUuid from the authentication token in the request.</li><li> Get it by using the https://oauth.izettle.com/users/me endpoint of OAuth2 API. See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) for more information.</li></ul> 
+|accountTypeGroup |string |path |required |The type of a merchant's Zettle account. You can use one of the following account types: <br/><ul><li> `PRELIMINARY` account where transactions are to be confirmed by third-party acquirers.</li><li> `LIQUID` account where transactions are to be paid out to the merchant.</li></ul>
+|start |string |query |required |A start time in UTC (inclusive) from when the transactions will be fetched. You can specify a UTC time in one of the following formats: <br/><ul><li>`YYYY-MM-DD` to specify a date. For example, `2020-11-29`.</li><li>`YYYY-MM-DDThh:mm:ss` to specify a time. For example, `2020-11-29T03:10:02`.</li></ul>
```

Comment:
This is not entirely true as i can for example get results for following request 
`/organizations/self/accounts/liquid/transactions?start=2021&end=2022` and it works perfectly fine.
My suggestion is to give a couple of most preferable examples OR to list all possible alternatives.

Reply:
same goes for  `end`